parts/k8s/cloud-init/artifacts/cse_config.sh

```diff
@@ -573,6 +584,7 @@ cleanUpContainerImages() {
 cleanUpGPUDrivers() {
   rm -Rf $GPU_DEST
   rm -f /etc/apt/sources.list.d/nvidia-docker.list
+  apt-key del $(apt-key list | grep NVIDIA -B 1 | head -n 1 | cut -d "/" -f 2 | cut -d " " -f 1)
```

Comment:
Adding this apt key cleanup here for non-GPU nodes (especially after discovering https://github.com/Azure/aks-engine/issues/3030)